src/main/resources/jira/atlassian-connect.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Synopsys Alert Plugin",
```

Comment:
Is this the plugin we upload? Or is the one below the plugin we upload?

Reply:
the atlassian-connect is the one we upload. We will have a different one in a bit and if you want specific changes, you will want to talk to Gavin.